### PR TITLE
SUP-15430: Fix spellchecker

### PR DESCRIPTION
**Pull request by guggi**
Issue: https://jira-gentics.atlassian.net/browse/SUP-15430

### CODE REVIEW

File: cms-core/src/main/resources/com/gentics/contentnode/runtime/config.yml [hunk 1]
```diff
@@ -114,6 +114,7 @@ aloha_plugins_global:
   - "common/commands"
   - "gcn/gcn"
   - "common/dom-to-xhtml"
+  - "gcn/spellcheck"
```
Comment (guggi) [inline cms-core/src/main/resources/com/gentics/contentnode/runtime/config.yml:117]: Maybe remove this to not have it activated as default?

Comment (deckdom) [reply cms-core/src/main/resources/com/gentics/contentnode/runtime/config.yml:117]: As far as I can tell, this isn't a special/paid/enterprise feature, and spellchecks are quite common, so having this enabled on default should be fine.

Regarding the change in the Aloha-Editor Repo:
Please add the entry for the `gcnSpellCheck` in here instead (Line `78`):

```yml
      - id: view
        merge:
          components: 'append'
        components:
          - 'gcnSpellCheck'
```

This should simply add the entry to the settings

**Review by deckdom — CHANGES_REQUESTED** [review]
Please see comment

File: cms-core/src/main/resources/com/gentics/contentnode/runtime/config.yml (hunk 1 above)
Comment (guggi) [reply cms-core/src/main/resources/com/gentics/contentnode/runtime/config.yml:121]: b57a6e7d

**Pull request merged**